agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/LazyHttpClient.java

```diff
@@ -19,7 +19,7 @@
  * DEALINGS IN THE SOFTWARE.
  */
 
-package com.microsoft.applicationinsights.agent.internal.httpclient;
+package com.azure.monitor.opentelemetry.exporter.implementation.utils;
```

Comment:
do we need LazyHttpClient in azure-monitor-opentelemetry-exporter?

Reply:
The quickpulse ping sender and data sender use httpclient to send quickpulse pings/data. Other alternative is to initialize quickpulse with httpclient. Do you recommend this approach?

Reply:
ya, if possible follow the approach of the exporter which only uses LazyHttpClient for agent side

Reply:
this will also allow us to not need the `AadAuthentication` classes I believe